### ISSUE TITLE
Suggestion: log seizure length

### ISSUE THREAD

**Issue by etjossem** [PebbleSeizureDetect/PebbleSeizureDetect#2]
For many epilepsy patients, it's important to record how long seizures last (to escalate response to convulsive status epilepticus, and also after the fact to aid diagnosis and treatment).

This app could definitely be extended to help do so. Long seizure-like events with no response from wearer could send a second set of text messages. And after the event, the Pebble might prompt the wearer/responder whether to log the duration, for automatic write to an equivalent of the [seizure diary](http://www.epilepsysociety.org.uk/free-epilepsy-smartphone-app).

**Comment by SlopeOak** [comment]
You are right; this could definitely be done.  But so far I'm not sure if any other people out there are using the app.  If it seems that people other than my wife are using it, I may make additions like this.  Or, if you would like to make the changes yourself, I would be happy to accept a patch.

Thanks!

**Comment by PebbleSeizureDetect** [comment]
You are right; this could definitely be done. But so far I'm not sure if any other people out there are using the app. If it seems that people other than my wife are using it, I may make additions like this. Or, if you would like to make the changes yourself, I would be happy to accept a patch.

Thanks!

**Comment by etjossem** [comment]
I don't have a Pebble, or I'd give it a shot!

(But just so you know, this app is the strongest reason I've seen so far to get one... :)

**Comment by TeMPOraL** [comment]
There's an [apparently successful company](http://smart-monitor.com/) literally selling overpriced Pebble watches bundled with a very similar app, so potential users are definitely there. This has a chance to turn into a free/Open Source alternative for those who already have a Pebble.
